api/envoy/api/v2/eds.proto

```diff
@@ -80,6 +81,12 @@ message ClusterLoadAssignment {
     //    "lb"_drop = 20%  // 50% of the remaining 'actual' load, which is 40%.
     //    actual_outgoing_load = 20% // remaining after applying all categories.
     repeated DropOverload drop_overloads = 2;
+
+    // Priority levels and localities are considered overprovisioned with this
+    // factor (in percentage). This means that we don't consider a priority
+    // level or locality unhealthy until the percentage of healthy hosts
+    // multiplied by the over provisioning factor drops below 100.
```

Comment:
So I totally created this problem* but can we decide if it's over-provisioned, overprovisioned, or over provisioned?  

I'm going to claim via wikipedia (https://en.wikipedia.org/wiki/Overprovisioning) it's one word, so we can try to do that and maybe rename this to overprovisioning_factor?

*
source/common/upstream/load_balancer_impl.h:// Priority levels and localities are considered overprovisioned
docs/root/intro/arch_overview/load_balancing.rst:Currently, it is assumed that each priority level is over-provisioned

Reply:
I believe I have checked in the fixes, but it's not shown here.